src/Extensions/Filters/PropertySelectionValueNameFilter.php

```diff
@@ -52,10 +53,41 @@ public function __construct(
     public function getFilters():array
     {
         return [
-            "propertySelectionValueName" => "getPropertySelectionValueName"
+            "propertyName"                  => "getPropertyName",
+            "propertySelectionValueName"    => "getPropertySelectionValueName"
         ];
     }
 
+    public function getPropertyName($property, $lang = null)
+    {
+        if ($lang === null)
+        {
+            $lang = $this->sessionStorageService->getLang();
+        }
+
+        if (is_null($this->propertyRepository))
+        {
+            $this->propertyRepository = pluginApp(PropertyRepositoryContract::class);
+        }
+
+        $propertyId = $property['propertyId'];
+        $propertyNames = $this->fromMemoryCache("propertyName.$propertyId", function() use ($propertyId, $lang)
+        {
+            return $this->authHelper->processUnguarded(function() use ($propertyId)
+            {
+                return $this->propertyRepository->findById($propertyId)->names;
+            });
+        });
+
+        $propertyName = $propertyNames->firstWhere('lang', $lang);
+        if(!is_null($propertyName))
```

Comment:
Wenn es keine Übersetzung in der $lang gibt, wird trotzdem ein leerer String ausgegeben im Frontend. Hier sollte auf leeren String geprüft werden. (Im SingleItem wird es auch mit leeren Texten ausgegeben)